packages/runtime-core/src/renderer.ts

```diff
@@ -82,10 +82,15 @@ export interface HydrationRenderer extends Renderer<Element | ShadowRoot> {
   hydrate: RootHydrateFunction
 }
 
+/**
+ * `boolean` is kept for backwards compatibility with the old `isSVG` attribute.
```

Comment:
This comment doesn't seem accurate anymore :)

Reply:
You're right. Removed it.